prog/minio/minio_server_nexus.rb

```diff
@@ -64,17 +64,28 @@ def cluster
   end
 
   label def bootstrap_rhizome
-    bud Prog::BootstrapRhizome, {"target_folder" => "minio", "subject_id" => vm.id, "user" => "minio-user"}
+    bud Prog::BootstrapRhizome, {"target_folder" => "minio", "subject_id" => vm.id, "user" => "ubi"}
 
     hop_wait_bootstrap_rhizome
   end
 
   label def wait_bootstrap_rhizome
     reap
-    hop_setup if leaf?
+    hop_create_minio_user if leaf?
     donate
   end
 
+  label def create_minio_user
+    begin
+      minio_server.vm.sshable.cmd("sudo groupadd -f -r minio-user")
+      minio_server.vm.sshable.cmd("sudo useradd -M -r -g minio-user minio-user")
```

Comment:
nit: I prefer verbose arguments. It's difficult to understand the intended behavior without referring to the manual.

```suggestion
      minio_server.vm.sshable.cmd("sudo groupadd -f --system minio-user")
      minio_server.vm.sshable.cmd("sudo useradd --no-create-home --system -g minio-user minio-user")
```